lib/source/pl/lib/std/pragmas.cpp

```diff
@@ -3,8 +3,24 @@
 #include <pl/core/validator.hpp>
 #include <pl/core/evaluator.hpp>
 
-namespace pl::lib::libstd {
+const long unsigned int LIMITMAX = std::numeric_limits<long unsigned int>::max() - 1;
+
+bool parse_limit(const std::string &value, long unsigned int *limit) {
+    char* s = (char*)value.c_str();
+    char** endptr = &s;
+    auto tlimit = strtol(s, endptr, 0);
```

Comment:
`std::stoll` might be the better alternative so you don't need to do this stuff right above